src/core/distillation/synth_aug_mmlu.py

```diff
@@ -0,0 +1,208 @@
+import os, ast, json
+from concurrent import futures
+
+import pandas as pd
+from tqdm import tqdm
+
+from core.utils.openrouter import openrouter
+from core.utils.chunker import chunker
+
+from core.prompts.mmlu_branches_aug import *
+
+# defaults
+DEFAULT_MODEL = os.getenv("OPENROUTER_MODEL", "deepseek/deepseek-r1:free")
+DEFAULT_MAX_TOKENS = int(os.getenv("OPENROUTER_MAX_TOKENS", "2048"))
+CHUNK_SIZE = int(os.getenv("SYNTH_CHUNK_SIZE", "16"))
+DUMP_EVERY = int(os.getenv("SYNTH_DUMP_EVERY", "10"))
+
+ALL_LETTERS = [chr(c) for c in range(ord("A"), ord("Z")+1)]
```

Comment:
Re-use what we have in https://github.com/LabARSS/reasoning-fine-tune/blob/85cc151cdfcac6a5ec409a9f2583486318fe7ed0/src/reasoning_fine_tune/prompts/mmlu_single_token_answer.py#L34? 
Extract it in a separate file for better readibility?

Reply:
@SemyonEpanov we alrady have `option_ids`